src/utils/file-helper.ts

```diff
@@ -0,0 +1,68 @@
+import { fromBuffer } from "file-type";
+
+export namespace FileHelper {
+	/**
+	 * formats bytes to bigger units
+	 */
+	export const bytesToSize = (bytes: number) => {
```

Comment:
this util is not needed on the backend and can be removed as well

(p.s. I just noticed it's also unused in FEE, was it intended to format the file size display?)

Reply:
yeah it was intended to render the file size but it appears not to be used so I have removed it in both repos